lib/node_modules/@stdlib/_tools/scaffold/math-iter-unary/scripts/scaffold.sh

```diff
@@ -147,7 +147,7 @@ unary_pkg_path="stdlib/math/base/special"
 unary_pkg="${unary_pkg_path}/${unary_pkg_alias}"
 
 # Define the REPL text main export description (note: 4 space indent and wrap at 80 characters):
-repl_text_main_desc=$(echo -n "${main_desc}" | "${wrap}")
+repl_text_main_desc=$(echo "${main_desc}" | "${wrap}")
```

Comment:
@gunjjoshi Why was the `-n` flag removed here? Having this suppresses appending a newline character to printed output.

Reply:
Keeping `-n` leads to an addition of `-n` in the output, something as follows:

```
-nReturns an iterator which iteratively computes the cube root.
```

Reply:
I see. The issue is the `-n` is not portable across all Linux shells.